extensions/svelte-web/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.5.0",
   "description": "Svelte language support for VS Code",
   "scripts": {
-    "build:grammar": "npx js-yaml syntaxes/svelte.tmLanguage.src.yaml > syntaxes/svelte.tmLanguage.json",
+    "build:grammar": "js-yaml syntaxes/svelte.tmLanguage.src.yaml > syntaxes/svelte.tmLanguage.json",
```

Comment:
As per [this comment](https://github.com/npm/npx/issues/9#issuecomment-786940691), `npx` requires manual input before it installs a package.

This is a problem for Gitpod prebuilds because prebuilds run in a headless environment. By adding `js-yaml` to the `devDependencies`, this problem is mitigated.